applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -71,6 +72,7 @@ static void ble_core_setup_random_static_addr(void)
 		 */
 	}
 }
+#endif
```

Comment:
```suggestion
#endif /* !defined(CONFIG_BT_PRIVACY) */
```

Same on line 193.
Hmm, I see that we are not consistent in if we use #endif comments either. This could be the same task as the one in my previous comment

Reply:
Forgot to add endif-comment, added it now